src/Razor/src/Microsoft.VisualStudio.DevKit.Razor/Telemetry/DevKitTelemetryReporter.cs

```diff
@@ -19,7 +19,7 @@ internal sealed class DevKitTelemetryReporter : TelemetryReporter
     private const string CollectorApiKey = "0c6ae279ed8443289764825290e4f9e2-1a736e7c-1324-4338-be46-fc2a58ae4d14-7255";
 
     [ImportingConstructor]
-    public DevKitTelemetryReporter() : base(telemetrySessions: [])
+    public DevKitTelemetryReporter() : base(telemetrySessions: ImmutableArray<TelemetrySession>.Empty)
```

Comment:
FYI, that I ended up fixing this in https://github.com/dotnet/razor/pull/9465 as well. (It's not clear to me how this sneaked into the product.)

Reply:
The compiler error that flags thid is very new. Like last week new.